### ISSUE TITLE
I get a blank png when use webshot function.

### ISSUE THREAD

**Issue by JiaxiangBU** [wch/webshot#72]
```
codelf <- "https://unbug.github.io/codelf/#ggplot"
library(webshot)
webshot(codelf,delay = 2,cliprect = 'viewport')
```

![webshot](https://user-images.githubusercontent.com/15884785/50590108-82105c80-0ec4-11e9-871b-a30546671d3e.png)

**Comment by sneumann** [comment]
Hi, what happens if you increase `delay` ? In my case rendering complex network widgets 
can take up to a minute. Yours, Steffen

**Comment by JiaxiangBU** [comment]
> Hi, what happens if you increase `delay` ? In my case rendering complex network widgets
> can take up to a minute. Yours, Steffen

Yes, I try 60 seconds for `delay`, but it is also a blank result.

Here is a reproducible minimal example.

```r
codelf <- "https://unbug.github.io/codelf/#ggplot"
library(webshot)
webshot(codelf,delay = 60,cliprect = 'viewport')
```

**Comment by wch** [comment]
@JiaxiangBU You can try out webshot2: https://github.com/rstudio/webshot2

**Comment by JiaxiangBU** [comment]
> @JiaxiangBU You can try out webshot2: https://github.com/rstudio/webshot2

Thanks, I try to install later in development version now.

https://github.com/rstudio/chromote/blob/75419863f553f3132756c9ee5b3fa5ab54a09ea4/DESCRIPTION#L17

**Comment by JiaxiangBU** [comment]
HI @wch I try this function `webshot2::webshot` but get an error, here is the issue https://github.com/rstudio/webshot2/issues/5
